regifter.sql

```diff
@@ -15,104 +15,114 @@ CREATE DATABASE regifter;
 -- giver - string
 -- value - integer
 -- previously_regifted boolean
-
+CREATE TABLE gifts (id SERIAL PRIMARY KEY, gift TEXT NOT NULL, giver TEXT, value INT, previously_regifted BOOLEAN);
 
 -- 
 \echo See details of the table you created
 -- 
-
+\dt 
 
 -- 
 \echo Alter the table so that the column price is changed to value 
 -- 
-
-
+--starting name is already value, removed and added to get practice in
+ALTER TABLE gifts DROP COLUMN value;
+--\d gifts;
+ALTER TABLE gifts ADD COLUMN value INT;
+--\d gifts
 -- 
 \echo Insert a peach candle, given by 'Santa' thats value is 9 and has been previously regifted
 -- 
-
-
+INSERT INTO gifts (gift, giver, previously_regifted, value) VALUES ('peach candle', 'Santa', TRUE, 9);
+--SELECT * FROM gifts;
```

Comment:
Same note - knock VALUES down to its own line 
Knock the data to its own line for each entry